cmd/yurthub/app/start.go

```diff
@@ -84,30 +81,10 @@ func NewCmdStartYurtHub(stopCh <-chan struct{}) *cobra.Command {
 
 // Run runs the YurtHubConfiguration. This should never exit
 func Run(cfg *config.YurtHubConfiguration, stopCh <-chan struct{}) error {
+	defer cfg.CertManager.Stop()
```

Comment:
Could we also use stopCh to stop CertManager?

Reply:
no, CertManager have stopCh that create by itself.